src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/entities/BaseCommandArguments.java

```diff
@@ -32,142 +35,7 @@
     private AbstractBuild<?, ?> build;
     private EnvVars envVars;
     private Logger logger;
-
-    public String getAppName() {
-        return appName;
```

Comment:
Why did you remove all of these params? Only due to lombok or something else?

Reply:
Just for lombok